indico/web/flask/app.py

```diff
@@ -153,7 +153,8 @@ def configure_emails(app, config):
     app.config['EMAIL_USE_TLS'] = config.SMTP_USE_TLS
     app.config['EMAIL_USE_SSL'] = False
     app.config['EMAIL_TIMEOUT'] = config.SMTP_TIMEOUT
-
```

Comment:
Don't remove the blank line; this now violates PEP8 (2 blank lines between top-level functions)